### PR TITLE
fix: Give default value for schedule workflow

### PR DESCRIPTION
**Pull request by litaocdl**
Give default value for schedule workflow

Closes: #1133 
Signed-off-by: Tao Li <tao.li@enterprisedb.com>

### CODE REVIEW

File: .github/workflows/continuous-delivery.yml [hunk 1]
```diff
@@ -498,7 +516,7 @@ jobs:
     runs-on: ubuntu-20.04
     env:
       # 'TEST_DEPTH' determines the matrix of E2E test cases that will be run.
-      TEST_DEPTH: ${{ needs.evaluate_options.outputs.test_level }}
+      TEST_DEPTH: ${{ needs.evaluate_options.outputs.depth }}
```
Comment (sxd) [inline .github/workflows/continuous-delivery.yml:519]: this it's test_level not depth, yeah I know that that confusing :P

Comment (litaocdl) [reply .github/workflows/continuous-delivery.yml:517]: Aha, OK, I see, we use different in python, Let me change back

Comment (litaocdl) [reply .github/workflows/continuous-delivery.yml:517]: seems from here, got it
```
// testDepthEnvVarName is the environment variable we expect the user to set
// to change the default test depth level
const testDepthEnvVarName = "TEST_DEPTH"
```

Comment (NiccoloFei) [reply .github/workflows/continuous-delivery.yml:517]: Fixed

**Review by NiccoloFei — APPROVED** [review]

**Review by hh24k — APPROVED** [review]

**Review by sxd — APPROVED** [review]

**Pull request merged**